ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,98 @@
+# Service Config
+# Port on which the service will listen to
+PORT={{sl_assessments_port | default("7201")}} 
+
+#Application ENV example local/dev/production
+NODE_ENV={{sl_assessments_node_env}}
+
+#Log configuration
+#Enable console logging
+ENABLE_DEBUG_LOGGING={{sl_assessments_enable_debug_logging | default("ON")}}
+
+#Enable writing logs to a file
+ENABLE_BUNYAN_LOGGING={{sl_assessments_enable_bunyan_logging | default("OFF")}}
+
+#service based URL
+APPLICATION_BASE_URL={{sl_assessments_application_base_url | default('/assessment/')}}
+
+# Allow report request to run for longer time
+request_timeout_for_reports={{sl_assessments_request_timeout_for_reports | default("600000")}}
+AUTHORIZATION={{sl_assessments_authorization}}
+MONGODB_URL={{sl_assessments_mongodb_url}}
+
+## Cloud config
+# Cloud Storage Configuration
+CLOUD_STORAGE={{sl_assessments_cloud_storage}}
+
+# Azure Cloud Configuration
+AZURE_ACCOUNT_NAME={{sl_assessments_azure_account_name}}
+AZURE_ACCOUNT_KEY={{sl_assessments_azure_account_key}}
+AZURE_STORAGE_CONTAINER={{sl_assessments_azure_storage_container)}}
```

Comment:
How will we support if I want to use AWS, I hope this app support aws also right?

One way I can think of is that, define a varible called `sl_assessment_cloud_config` in `defaults/main.yaml` and use that variable here. For example,

```
# defaults/main.yaml

sl_assessment_cloud_config: |
  AZURE_ACCOUNT_NAME={{sl_assessments_azure_account_name}}
  AZURE_ACCOUNT_KEY={{sl_assessments_azure_account_key}}
  AZURE_STORAGE_CONTAINER={{sl_assessments_azure_storage_container)}}
  # You can add commented aws example configs here. So that if anyone wants to use that,
  # They can uncomment this one, and comment above in their common.yml file
```
```
#  ansible/roles/stack-sunbird/templates/sl-assessments.env
# Here you can add something like this
{{ sl_assessment_cloud_config }}

# which will be templatized from the above variable.
```

Reply:
i have defined like this in env
## Cloud Environment platform
# which will be templatized from the above variable (defaults)
cloud_config={{ sl_assessment_cloud_config}} 

i have added in defaults like this
#################################### Cloud config ########################################
sl_assessment_cloud_config: |
# Azure Cloud Configuration
AZURE_ACCOUNT_NAME={{sl_assessments_azure_account_name}}
AZURE_ACCOUNT_KEY={{sl_assessments_azure_account_key}}
AZURE_STORAGE_CONTAINER={{sl_assessments_azure_storage_container)}}


## Google Cloud Configuration
# GCP_PATH={{sl_assessments_gcp_path}}
# GCP_BUCKET_NAME={{sl_assessments_gcp_bucket_name)}}


## AWS Cloud Configuration
#AWS_ACCESS_KEY_ID={{sl_assessments_aws_access_key_id}}
#AWS_SECRET_ACCESS_KEY={{sl_assessments_aws_secret_access_key}}
#AWS_BUCKET_NAME={{sl_assessments_aws_bucket_name}}
#AWS_BUCKET_ENDPOINT={{sl_assessments_aws_bucket_endpoint}}
#AWS_BUCKET_REGION={{sl_assessments_aws_bucket_region}}

Reply:
i have mentioned like this is env
## Cloud Environment platform
# which will be templatized from the above variable (defaults)
cloud_config={{ sl_assessment_cloud_config}}

Reply:
changes done rajesh kindly look in to that and update me

Reply:
@sivakarthikeyanoptit - I will share key names for other cloud config

Reply:
indentation corrected